oasis-webapp/src/main/resources/templates/messages_bg.xlf

```diff
@@ -3,8 +3,8 @@
   <file original="SoyMsgBundle" datatype="x-soy-msg-bundle" xml:space="preserve" source-language="en-GB" target-language="bg-BG">
     <body>
       <trans-unit id="85256968653381481" datatype="html">
-        <source>https://www.ozwillo.com/en/terms</source>
-        <target>https://www.ozwillo.com/bg/uslovia</target>
+        <source>https://www.ozwillo.com/en/legal-notices</source>
```

Comment:
That's not how it works actually: the `id` depends on the `source` (and _meaning_; changing the `source` changes the `id`. It works very similar to [Angular i18n](https://angular.io/guide/i18n), except we cannot set custom IDs (and select/plurals use a different syntax)